### ISSUE TITLE
Please, fix the mountains.

### ISSUE THREAD

**Issue by NigelFarage666** [EarthMC/Issue-Tracker#262]
Mountains aren't high enough and there is almost no snow at their top and I can even find sand its kind of sad that in this time of the year I can barely find ANY snow on the Alps and Himalayas.
Second problem is costume trees which everyone hates because of whom we can't see mountain tops.
Mountain shapes are completely weird and should we fixed and more sharp. 
Mountains are not high enough.
There are some mountains missing/not high enough at all.

1.Alps
-Add more snow
-Remove some costume trees who are close to the mountains or on them and increase mountain height in the areas you wish them to incrase. 

 
![alps](https://user-images.githubusercontent.com/44730188/50562980-5b61f280-0d19-11e9-8621-b52147a35244.png)

YELLOW= Snow 
RED= Marking Alps

![alps](https://user-images.githubusercontent.com/44730188/50563012-a976f600-0d19-11e9-934b-5ca10fd56055.png)

2.Himalaya Mountains

Himalayas suffer from similar issues.
Mountain range is really thin and small and there is barely any snow.
my suggestions are.

-Increase amount of snow.
-Incrase Mountain height on certain areas
-Mountains looks generally poorly shaped I suggest you make them more sharper on the top.

![himalayas](https://user-images.githubusercontent.com/44730188/50563129-90bb1000-0d1a-11e9-964d-06ef0153db1f.jpg)

3.Pyreenes 

Well this mountain range is done pretty well I just have 2 complaints.
-There is no Ebro river which must be fixed 
-There should be more snow and less trees.
![spain](https://user-images.githubusercontent.com/44730188/50563159-c4963580-0d1a-11e9-9ab8-f123117f7c71.jpg)

4.Zagros (Persia)

This mountain is almost high as Alps and still barely made on the map. 
Its not even close to Alps in terms of height
-Increase mountain height (drastically)
-Place some snow on it.

![persian mountain ranges](https://user-images.githubusercontent.com/44730188/50563221-35d5e880-0d1b-11e9-8f88-72af7055aef1.jpg)

5.Caucasus Mountain Range. 

-More snow needed
-Remove Oak Trees and place some more Spruce Trees.

6.Rocky Mountains.

There are too many trees and they are simply not high enough. 

By sharp top of the mountain I meant this. It does not need to look the same it is just an example of my thought. If you remember Classic Mountain tops you know what I mean.
![minecaft mountain](https://user-images.githubusercontent.com/44730188/50566306-552f3e80-0d38-11e9-8ee8-c7e364031cc1.png)

Note:
Ones who will assign this job for themselves DM me on discord for more info If needed and I welcome all suggestions. Please do this before people settle on those mountains!
My discord is:   NigelFarage#3781

Thank you for your help!

**Comment by Sulakek** [comment]
I climbed Mount Everest i even think it's quite short compared to other mountains and also The Himilayas are ugly cause they have sand in between with weird deadbushes near it, and theres not enough snow too.

**Comment by NigelFarage666** [comment]
I agree they are very ugly and short.

I can understand how big map was and I appreciate Teams work.

**Comment by PoppyK** [comment]
!.  I have walked the Himalayas many times and flew over them today.  Mt. Everest is at Y 195 which is quite high.  The entire range is in proportion to Everest.  I don't feel any height needs to be added, nor width as there is not much base area to widen it.  They also have plenty of snow.  No editing is needed on the  Himalayas.  The pics show aerial height and depth:
https://i.gyazo.com/abd890db9e5c12922aa2d8fefd23f573.png https://i.gyazo.com/d8a0bd0dec993f78189fa3ab735fce10.png

2. The Alps.  Again they are comparable in size and proportionate to other mountains.  I used this link to help with the altitude of mountain ranges.  https://en.wikipedia.org/wiki/List_of_mountains_by_elevation   Mount Blanc, the highest range in the Alps is at y 146.  I did add snow to the peaks of most of the mountains.  That was the only editing done.  
 
3.  I flew over the Andes briefly.  But based on looking at these three mountain ranges alone, I don't think editing needs to or should be attempted.  We have some amazing builds on top of these mountains as well as memorials.  We don't want to interfere with what has already been built.  

What you are proposing could potentially take hours and hours of work.  It took me a month of work to form the grand canyon, which was necessary because there wasn't one present on the map.  But I don't feel the mountains need to be edited in any drastic way.  They might not be to your specifications of what you wanted to see but I think the mapmaker did a nice job of being proportionate to real life.  And realistically, many of the mountain tops will be or have been claimed and terraformed to allow for towns to be added.  

At this point, I feel that designers should concentrate on things that will benefit trade and travel, not purely aesthetics.
